Add Markdown translation of NumFOCUS CoC

stefanv opened the pull request:
For including in their own projects, I imagine projects would find a Markdown version of our CoC handy.  I've translated ours from the Google Doc, and removed the section about "what if this happens at a NumFOCUS event", since that is not relevant for projects.

stefanv committed:
Add Markdown translation of NumFOCUS CoC

stefanv committed:
Wrap unwrapped paragraphs

stefanv committed:
Reformat licensing line

Dr-G commented:
I can't guarantee that the word doc is identical to what's on the NumFOCUS website. We should treat the webpage as canonical for the CoC. https://numfocus.org/code-of-conduct

It would be pretty simple to download the webpage as a file and then do a find-replace on html tags to convert them to markdown. Let me know if I can be helpful here.

Dr-G commented:
I would also suggest leaving in the part about events, since many projects do hold their own events (i.e. ProjectCon)

stefanv commented:
The translation isn't that straightforward because of heading levels etc., but you're welcome to give it a shot.

The problem with that section is that it refers specifically to *NumFOCUS events*.

Dr-G commented:
Ok, so to confirm I'm understanding you: the idea is that projects could replace "NumFOCUS" with their project name and effectively adopt the language of the CoC, but not the reporting structures, etc?

stefanv commented:
There's some ambiguity to be resolved here.  NumFOCUS has its own CoC for its own purposes, but it is also the CoC recommended for *external* projects.  However, the wording is from the perspective of NumFOCUS, not an external project.  So we may have to adjust the CoC a bit to make it suitable for external use.

Dr-G committed:
Update numfocus-coc.md

Made edits so that this is appropriate for projects to adopt directly by replacing [BRACKETED] information with their project information.

Dr-G committed:
Update numfocus-coc.md

Missed the very top line to replace "NumFOCUS" with [PROJECT]

Dr-G commented on manual/numfocus-coc.md:
```diff
@@ -0,0 +1,238 @@
+
+Thank you for helping make this a welcoming, friendly community for all.
+
+**CODE OF CONDUCT REPORTING FORM:** [*https://numfocus.typeform.com/to/ynjGdT*](https://numfocus.typeform.com/to/ynjGdT)
```
Projects should host their own reporting form, since they would be running their own CoC adjudication process.

Dr-G commented on manual/numfocus-coc.md:
```diff
@@ -0,0 +1,238 @@
+Thank you for helping make this a welcoming, friendly community for all.
+
+**CODE OF CONDUCT REPORTING FORM:** [*https://numfocus.typeform.com/to/ynjGdT*](https://numfocus.typeform.com/to/ynjGdT)
+
```
I strongly suggest creating a Table of Contents at this point (just after linking the reporting form) if possible. It helps a lot for people to be able to jump directly to the portion of the CoC they want to review, since otherwise it is a fairly daunting wall of text. See here (https://numfocus.org/code-of-conduct) for the example.

stefanv replied:
Thanks, Gina!  I added a table of contents.

stefanv committed:
Add table of contents

stefanv committed:
Remove links to NumFOCUS reporting form; remove NumFOCUS privacy policy

stefanv commented:
@Dr-G I removed the privacy policy link, since I can't think of any projects that has its own.  I also modifed the typeform links to indicate that they should be added by the project.
